blog/technical/articles/redis-pub-sub.md

```diff
@@ -0,0 +1,50 @@
+[Home](../../../README.md) > Blog > [Technical](../blog-technical.md) <!-- Breadcrumb -->
+
+# Redis PubSub
+
+## The problem with scaling and the cloud
+
+This project was meant to be deployed to the cloud, but what has this to do with this topic?<br>
+The cloud allows us to start and stop instances of our micro services in just a few seconds.
+
+This clearly is a great advantage of the cloud, but what if we want to communicate between different instances? In reality it's hard to know the IP of every currently running micro service to try to communicate with them.<br>
+Micro services in a cloud start and stop (sometimes crash) without any interaction from our side.
+
+
+We also have the problem that not every instance wants to know everything currently happening in our infrastructure. Sending notifications to these micro services would mean alot of overhead which increases dramastically every time our infrastructure scales itself up.
+
+Because of that we need a _"central"_ instance knowing:
+
+* The IPs of every micro service and
+* What every instance really wants to know
+
+So, we need another micro service, right? No.<br>
+A single micro service handling all the things we want to share with other micro services, would mean we have to develop and maintain it.<br>
+Even if we would maintain it: What happens if the service itself crashes or gets slow because of millions of messages per minute? This would mean that we wouldn't be able to communicate between our micro services which could lead to 1. data loss and 2. unavailable features or even 3. broken security mechanisms.
```

Comment:
Correction = "1. data" -> "1. Data"
Correction = "2. unavailable" -> "2. Unavailable"
Correction = "3. broken" -> "3. Broken"

Is this better ?